bluesky/simple_scans.py

```diff
@@ -66,8 +67,8 @@ class _BundledScan:
 
     def __init__(self):
         # subs and sub_factories can be set individually per instance
-        self.subs = self.default_subs
-        self.sub_factories = self.default_sub_factories
+        self.subs = list(self.default_subs)
```

Comment:
As a reminder: the callbacks that don't need Events really shouldn't be called on event Event, which is why we accept dicts. On the other hand, forcing users to understand the Document Model just to add a table seems unnecessary. Since our built-in callbacks can safely accept every document, we can afford to be flexible and guess.